api/models.py

```diff
@@ -28,22 +29,31 @@ def __str__(self):
 
 
 class Course(models.Model):
+    def __str__(self):
+        return self.num
+
     num = models.IntegerField(default=0, primary_key=True)
     name = models.CharField(max_length=200)
     credit = models.IntegerField(default=0)
     prof_name = models.ForeignKey(Professor, on_delete=models.CASCADE)
     classroom = models.CharField(max_length=200)
-    weekday = models.CharField(max_length=100)  # 강의 요일
-    start_time = models.TimeField(auto_now=False, auto_now_add=False)  # 강의시간
-    finish_time = models.TimeField(auto_now=False, auto_now_add=False)
+    weekday = models.CharField(max_length=100, default='')  # 강의 요일
+    start_time = models.TimeField(auto_now=False, auto_now_add=False, null=True)  # 강의시간
+    finish_time = models.TimeField(auto_now=False, auto_now_add=False, null=True)
 
 
 class Basket(models.Model):
+    def __str__(self):
+        return self.course_num
+
     course_num = models.ForeignKey(Course, on_delete=models.CASCADE)
     std_id = models.ForeignKey(Student, on_delete=models.CASCADE)
```

Comment:
여기도 위에서 드린 피드백과 마찬가지로 필드명을 `student`로 바꾸는 게 어떨까 싶습니다🐵

```suggestion
    student = models.ForeignKey(Student, on_delete=models.CASCADE, related_name='baskets')
```